app/views/professors/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h3><%= @professor.name %>'s Students<p></h3>
+
+<% @professor.students.each do |student| %>
+  <%= student.name %><br>
+<% end %>
+
+Average age of Students: <%= @professor.students.average_student_age %><br>
```

Comment:
See comment above about considering which model specific methods should be defined in. I start to feel like my objects are a little too coupled together when I see more than two methods chained in a row. It feels like the [law of demeter](https://betterprogramming.pub/demeters-law-don-t-talk-to-strangers-87bb4af11694) is being violated and professors knows a little too much about the student model now.